new-backoffice/extending/package-manifest/README.md

```diff
@@ -0,0 +1,146 @@
+    "version": "",
+    "allowPackageTelemetry": true,
+    "extensions": []
+}
+```
+
+### Telemetry elements
+
+In version 9.2 some additional root elements were added. The purpose of these are to control and facilitate telemetry about the package, none of these are mandatory. The properties are:
+
+-   `name` - Allows you to specify a friendly name for your package that will be used for telemetry, if no name is specified the name of the folder will be used instead
+-   `version` - The version of your package, if this is not specified there will be no version specific information for your package
+-   `allowPackageTelemetry` - Allows you to entirely disable telemetry for your package if set to false, defaults to true.
+
+Example `umbraco-package.json` with telemetry elements:
+
+```json
+{
+    "name": "My Awesome Package",
+    "version": "1.0.0",
+    "allowPackageTelemetry": true
+}
+```
+
+## Extensions
+
+The `extensions` collection is an array of extension objects, each object describes a single extension. The `type` property is mandatory, and must be one of the following values:
+
+-   `entryPoint`
+-   `collectionView`
+-   `dashboard`
+-   `dashboardCollection`
+-   `entityAction`
+-   `entityBulkAction`
+-   `headerApp`
+-   `healthCheck`
+-   `itemStore`
+-   `menu`
+-   `menuItem`
+-   `modal`
+-   `packageView`
+-   `propertyAction`
+-   `propertyEditorModel`
+-   `propertyEditorUI`
+-   `repository`
+-   `section`
+-   `sectionSidebarApp`
+-   `sectionView`
+-   `store`
+-   `theme`
+-   `tree`
+-   `treeItem`
+-   `treeStore`
+-   `userProfileApp`
+-   `workspace`
+-   `workspaceAction`
+-   `workspaceEditorView`
+-   `workspaceViewCollection`
+
+## A special note about `entryPoint`
+
+The `entryPoint` extension type is special, it is used to specify the entry point for a package. This is the only extension type that can be used to specify the entry point for a package. An entry point is a single javascript file that will be loaded when the Backoffice starts. This is useful if you want to load a single javascript file that will then load all the other files for your package. You can apply any conditional logic when registering your entry point, for example you can check if the user is logged in, or if the user has access to a specific section before registering additional extensions.
+
+This is also the way to go if you want to load in external libraries such as jQuery, Angular, React, etc. You can use the `entryPoint` to load in the external libraries to be shared by all your extensions. You can also load global CSS files in the `entryPoint` extension.
+
+Read more about the `entryPoint` extension type in the [Entry Point](./entry-point.md) article.
+
+## Bundling
+
+{% hint style="info" %}
+Since version 14.0.0 Umbraco will not automatically try to bundle the static assets of packages. This is to avoid any potential conflicts with other bundling packages and to allow for more flexibility in how you want to bundle your assets.
```

Comment:
```suggestion
From version 14.0.0 Umbraco will not automatically try to bundle the static assets of packages. This is to avoid any potential conflicts with other bundling packages and to allow for more flexibility in how you want to bundle your assets.
```
Little grammar tweak 🙂 

Or "Umbraco versions 14.0.0 and above will not automatically...." might read nicer.